pdf.js

```diff
@@ -258,6 +262,44 @@ function PDFViewer(browserApi) {
 
   // Request translated strings.
   cr.sendWithPromise('getStrings').then(this.handleStrings_.bind(this));
+
+  // Setup global functions for scripting from electron
+  global.fitToWidth = function() {
```

Comment:
We have the pdf viewer instance available for scripting with `window.viewer`, its better to use it in Electron as required for testing, can help reduce the changes to this fork. Thoughts ?